openbb_platform/obbject_extensions/charting/openbb_charting/__init__.py

```diff
@@ -44,19 +57,19 @@ def __init__(self, obbject):
         """Initialize Charting extension."""
         # pylint: disable=import-outside-toplevel
         from openbb_core.app.model.charts.charting_settings import ChartingSettings
-        from openbb_core.app.model.obbject import OBBject
 
         self._obbject: OBBject = obbject
         self._charting_settings = ChartingSettings(
             user_settings=self._obbject._user_settings,  # type: ignore
             system_settings=self._obbject._system_settings,  # type: ignore
         )
-        self._handle_backend()
```

Comment:
We should keep this in the constructor instead of having other functions (which responsibility is different) doing it.

Reply:
This is to prevent it from running without being asked to.